aten/src/ATen/native/Convolution.cpp

```diff
@@ -413,8 +413,19 @@ struct ConvParams {
     if (!detail::getCUDAHooks().compiledWithCuDNN()) {
       return false;
     }
+    static long cudnn_version = detail::getCUDAHooks().versionCuDNN();
+    // broken on cuDNN 9.8
+    if (cudnn_version >= 90800) {
+      if (cudnn_conv_suggest_memory_format(input, weight) == at::MemoryFormat::Contiguous &&
+          input.scalar_type() == at::kBFloat16 || input.scalar_type() == at::kHalf) {
+        for (int i = 2; i < weight.dim(); i++) {
```

Comment:
does it affect both 2d and 3d, or 3d only?

Reply:
I'll check with the team to confirm but my understand is it's 3d only as the user who reported said they could work around it with a 2D conv

Reply:
so you need to also check `weight.dim() == 5`?